src/flb_network.c

```diff
@@ -1000,7 +1000,7 @@ static struct flb_dns_lookup_context *flb_net_dns_lookup_context_create(
      *                 the number of retries to 2
      */
 
-    optmask = ARES_OPT_FLAGS;
+    optmask = ARES_OPT_FLAGS | ARES_OPT_TRIES;
```

Comment:
Could you please remove the flag and the following line?
After some consideration I remembered that due to a design constraint fluent-bit won't allow c-ares to retry so we'll have to wait until that is worked on before we can enable retries.

Reply:
@bazzargh ping

Reply:
Sorry I just got back from a holiday watching the eclipse! Updated as requested.